cmd/root.go

```diff
@@ -117,9 +118,14 @@ func serve() error {
 		log.Fatal("Can't find backend:", plugin)
 	}
 
+	// Build Alerts Object
+	alerts := alerts.Alerts{db, []alerts.Alert{}}
```

Comment:
@yaacov so I have added a very simple PR which creates an Alerts object with a `db` and an empty list and then calls `Open` which does nothing.

Reply:
Next step is to add a PR which parses the config file for alerts and passes it to `Open` as an options object.